### ISSUE TITLE
Use LAST year as a test set for prediction, keep it out of training set

### ISSUE THREAD

**Issue by SJCaldwell** [mackenziedg/UFOTracker#4]
use up through year 2015 as a training set, then use 2016 as a test set. 

(May not be 2015 - double check what our last common year is for all variables and save that one)

**Comment by fbchow** [comment]
Side Note:
Something to be cognizant of:

Domain of time for each dataset could have affected whether or not the variables were significant enough to be included in the model.  

- breweries starts in 1984, while the other data starts in 1974. That's why there are a bunch of NA's in breweries for any year before 1984.

- internet rurality only goes from 2000-2015

- per capita gdp only goes up to 2014

- census data for population per state only contains every 10 years: "1970", "1980", "1990", "2000", "2010"


The **last common year was 2014** (_from per capita gdp)._ So perhaps we can try to fit 2012 or 2013 and cross-validate with year 2014. 

The **first common year that all the datasets share is 2000** (_due to internet rurality csv_)
